README.md

```diff
@@ -199,6 +199,17 @@ token was obtained:
 - `X-Token` header containing the new access token to be used in future requests.
 - `X-Refresh-Token` header containing the new refresh token.
 
+You may configure some code to be ran (scoped to a controller) when a token is
```

Comment:
Awesome to include this in the README! :+1: I think this should be `to be run`, though.